lms/templates/registration/account_activation_sidebar_notice.html

```diff
@@ -0,0 +1,32 @@
+<%page expression_filter="h"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<div class="profile-sidebar" role="region" aria-label="Account Activation Info">
+  <header class="profile">
+    <h2 class="account-activation-title sr">${_("Account Activation Info")}: </h2>
+  </header>
+  <div class="user-info">
+    <ul>
+      <li class="status status-verification warning" role="alert" aria-labelledby="title status-title" tabindex="-1">
+        <span class="title status-title">
+          <i class="fa fa-exclamation-circle" aria-hidden="true"></i>
+          ${_("Check your email!")}
+        </span>
+        <p class="status-note">${Text(_(
+          "To sign back into your {platform_name} account,"
+          "check your {email_start}{email}{email_end} inbox for an account activation "
+          "link from {platform_name}."
+        )).format(
+          email_start=HTML("<strong>"),
+          email_end=HTML("</strong>"),
+          email=email,
+          platform_name=platform_name,
+        )}
+        </p>
+        <a class="btn btn-neutral"><i class="fa fa-envelope-o"></i> Resend Activation Email </a>
```

Comment:
This button does not appear to do anything at present. Should it, or are we deferring that to a later story? If we're deferring, then we should definitely wait on enabling this until it's implemented.